sycl/include/sycl/ext/intel/experimental/esimd/common.hpp

```diff
@@ -176,11 +176,279 @@ enum class atomic_op : uint8_t {
   /// Compare and exchange (floating point).
   /// <code>if (*addr == src0) *addr = src1;</code>
   fcmpwr = 0x12,
+  fadd = 0x13,
+  fsub = 0x14,
+  load = 0x15,
+  store = 0x16,
   /// Decrement: <code>*addr = *addr - 1</code>. The only operation which
   /// returns new value of the destination rather than old.
   predec = 0xff,
 };
 
+/// The scope that lsc_fence operation should apply to
+/// Supported platforms: XeHP_SDV, DG2, PVC
```

Comment:
DG2 is the first platform with LSC:
```suggestion
/// Supported platforms: DG2, PVC
```

Reply:
Done.